doc/hardware/at86rf212.rst

```diff
@@ -0,0 +1,105 @@
+.. at86rf212:
+
+AT86RF212 setup
+##################
+
+Category: radio
+Standard: IEEE 802.15.4
+Boards: MinnowBoard MAX, Galileo Gen 2
+
+Connection with MinnowBoard MAX board
+
+
+.. image:: _static/at86rf212-minnow.png
+
+Connection with Galileo Gen 2 board
+
+
+.. image:: _static/at86rf212-galileo.png
+
+
+.. table:: Pin table of at86rf212 radio board:
+
+    =====  ==========================
+    PIN    Name
+    =====  ==========================
+    1    	SLP_TR
+    2    	CLKM
+    3    	DIG2
+    4    	VCC
+    5    	DIG1
+    6    	VCC
+    7    	SCLK
+    8    	GND
+    9    	nRST
+    10    	GND
+    11    	IRQ
+    12    	MISO
+    13    	DIG4
+    14    	MOSI
+    15		DIG3
+    16		NSEL
+    =====  ==========================
+
+
+.. table:: Connect at86rf212 board to MinnowBoard MAX low speed expansion connector (LSE):
+
+    ============  ==========================
+    at86rf212 		LSE
+    ============  ==========================
+    PIN1(SLP_TR) 	PIN23
+    PIN6(VCC3.3)	PIN4
+    PIN7(SCLK)		PIN11
+    PIN9(nRST)		PIN21
+    PIN10(GND)		PIN2
+    PIN11(IRQ)		PIN22
+    PIN12(MISO)		PIN7
+    PIN14(MOSI)		PIN9
+    Pin16(CS)		PIN5
+    ============  ==========================
+
+
+.. table:: Connect at86rf212 board to Galileo Gen 2 board:
+
+    ======================  ==========================
+    at86rf212 			Galileo Gen 2
+    ======================  ==========================
+    PIN1(SLP_TR) 		IO2
+    PIN6(VCC3.3)		VCC3.3
+    PIN7(SCLK)			IO13
+    PIN9(nRST)			IO1
+    PIN10(GND)			GND
+    PIN11(IRQ)			IO3
+    PIN12(MISO)			IO12
+    PIN14(MOSI)			IO11
+    Pin16(CS)			IO10
+    ======================  ==========================
+
+
+Note: For MinnowBoard MAX, after finish the above hardware setup, please check LPSS & SCC Configuration in BIOS, to make sure the LPSS SPI support is enabled.
+
+After finish the hardware setup, please boot into the system and perform the below steps to enable the kernel module auto loading:
+ln -s /lib/modprobe.d/at86rf230-minnow.conf.sample /etc/modprobe.d/at86rf230-minnow.conf
+ln -s /usr/lib/modules-load.d/at86rf230-minnow.conf.sample /etc/modules-load.d/at86rf230-minnow.conf
+
+Reboot the board, the related device driver and kernel modules will be auto loaded, then execute these commands to configure:
+
+On device1:
+
+#. `ip link set wpan0 address a0:0:0:0:0:0:0:1`
+#. `iz set wpan0 777 8001 11`
+#. `ifconfig wpan0 up`
+#. `ip link add link wpan0 name lowpan0 type lowpan`
+#. `ifconfig lowpan0 up`
```

Comment:
err... what happened with the systemd service and scripts you submitted before? They should replace these commands! and those on the other .rst as well